### PR TITLE
[crashpad] use vcpkg_cmake_get_vars

### PR DESCRIPTION
**Pull request by Neumann-A**
probably superseded by #23827

### CODE REVIEW

File: ports/crashpad/vcpkg.json [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
-  "port-version": 1,
+  "port-version": 2,
   "description": [
     "Crashpad is a crash-reporting system.",
     "Crashpad is a library for capturing, storing and transmitting postmortem crash reports from a client to an upstream collection server. Crashpad aims to make it possible for clients to capture process state at the time of crash with the best possible fidelity and coverage, with the minimum of fuss."
   ],
   "homepage": "https://chromium.googlesource.com/crashpad/crashpad/+/master/README.md",
   "license": "Apache-2.0",
-  "supports": "osx | windows",
+  "supports": "osx | (windows & !uwp)",
```
Comment (Adela0814) [inline ports/crashpad/vcpkg.json:11]: Can you explain why `!uwp` was added?

**Review by Adela0814 — DISMISSED** [review]

File: ports/crashpad/vcpkg.json (hunk 1 above)
Comment (Neumann-A) [reply ports/crashpad/vcpkg.json:11]: https://dev.azure.com/vcpkg/public/_build/results?buildId=75392&view=logs&jobId=afe306fb-88a0-5d5e-2934-71aa01507316&j=afe306fb-88a0-5d5e-2934-71aa01507316 failed

One of the errors is:
../src/86e8d5a126-0d522bae5c.clean/client/crashpad_client_win.cc(828): error C3861: 'GetVersion': identifier not found
 According to 
 https://docs.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getversion
 This is only supported for desktop apps. It also uses `NtCreateThreadEx` and other stuff which certainly is not supported by store apps.

Comment (Neumann-A) [reply ports/crashpad/vcpkg.json:11]: It only build before since it used the normal windows flags instead of the correct UWP flags.

**Comment by BillyONeal** [comment]
Thanks for the fix!

**Pull request merged**